[ADD] general/payment_acquirers: add doc on Adyen manual capture (#1153)
odoo/documentation: content/applications/general/payment_acquirers/adyen.rst

   :align: center
   :alt: Allows payments originated from a specific domain.

Capture payments after the delivery
-----------------------------------

> Adyen allows you to capture an amount manually instead of having an immediate capture.

Author: I'll keep `immediate`, not because I find it clearer, but because that's how this choice is called in Adyen's Backend

To set it up, enable the **Capture Amount Manually** on Odoo, as explained in the :ref:`payment
acquirers documentation <payment_acquirers/capture_amount>`, then on Adyen.

To do so, open your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
**Capture Delay** to **manual**.

.. image:: media/adyen_capture_delay.png
   :align: center
   :alt: Capture Delay settings in Adyen

.. warning::

>    * If you set the **Capture Amount Manually** on Odoo but not on Adyen, Odoo won't detect it.
>      Your authorization request would be automatically approved in an immediate capture on Adyen's
>      side.

Reviewer: here's yet another suggestion for this paragraph, I changed the tenses so that there's not future tense used here

```suggestion
   * If you set the **Capture Amount Manually** on Odoo but not on Adyen, Odoo can't detect it.
     Therefore, your authorization request is automatically approved in an immediate capture on 
     Adyen's side.
```

>    * If you set the **Capture Amount Manually** on Odoo but not on Adyen, Odoo can't detect it.
>      Therefore, your authorization request would be automatically approved in an immediate capture
>      on Adyen's side.

Reviewer 2: Although correct, it's a rather technical explanation. In the end, all the user needs to know is "proper configuration = good; otherwise bad".
```suggestion
   * If the **Capture Delay** is not set to **manual** on Adyen, Odoo will ignore the configuration option **Capture Amount Manually**.
```
Please correct if it's not what will actually happen on Odoo.

Author: That's False, Odoo will not ignore it, and since the `Authorisation` notification will be exactly the same, Odoo will put the transaction as `authorized` where Adyen will consider it `done` and never touch it again. And so, Odoo will wait... and wait... and wait... And every capture or void request will just return with a `success: false`. That's why I put an emphasis here. This'll break every tx if not done correctly.

Reviewer 2: Ok. Then,
1. Make it a `danger` admonition.
2. Change the line to something like "If the **Capture Delay** is not set to **manual** on Adyen but Odoo is not configured to capture amounts manually, it will be impossible to capture or cancel payments from Odoo."
3. Split the second line into another admonition, a `note` should be fine IMO.
4. Use dashes (`-`) for bullet points.

Reviewer 3: I may have misread this the first time because I didn't understand the _dangerous_ aspect of it. A good admonition would work marvelously here indeed :)

   * After **7 days**, if the authorization hasn't yet been captured, the customer has the right to

>      **revoke** it.

Reviewer 2: Like, for real? I thought it was automatic for all providers. How would the customer do that from their side?

Author: They call their bank, send a request from their bank app... Actually, this may be better if we could put it on the `payment_acquirers` page, if we can confirm the delay is the same for all acquirers. But yes, I can definitely do so from my bank app.
Thing is, the provider Stripe (or Authorized?), for instance, specify that an authorization is automatically revoked after 30 days if it's not captured/void yet, Adyen don't specify this, but instead speaks of the 7 days for the client, which the other provider don't mention.
I think the actual delay(s ?) might depend more on your bank than on the provider, but I'm not sure. Either way, I feel this should be written somewhere.

Author: omg, I didn't test this notification in ma task... Is it a normal void notification? I hope it's a normal void notification... :fearful:

Reviewer 2: I think it's fine to only mention it in Adyen doc for now, especially if it depends on the provider.

However, you capture a payment (or a transaction), not an authorization.

   :align: center
   :alt: Allows payments originated from a specific domain.

Capture the payment after the delivery
--------------------------------------

If you wish to manually capture an amount instead of having an immediate capture, Adyen allows it.

You have to enable it both on Odoo, as explained :ref:`here <payment_acquirers/capture_amount>`,

> and on Adyen.

Reviewer 3: @LoredanaLrpz 's suggestion is good but I would also request to have the link on the relevant text/title, rather than on "here". it helps the reader to know directly what the link is about, and it is great for SEO improvement.

> To do so, on your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
> **Capture Delay** to **manual**.

Reviewer 3: ^^
```suggestion
To do so, open your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
**Capture Delay** to **manual**.
```

> .. image:: media/adyen_capture_delay.png

Reviewer 3: Media file names use hyphens and are in a distinct media folder, named as the doc's filename.
Please refer to the [Content guidelines: images](https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#media-files) for more information.
If needed, you can also move previous images to that new folder

Author: Since I messed it up when I rewrote the payment acquirers documentation (this PR was written before, seemed like I merged unused file... my bad...), this image is already there. I could change just this image now, but we'll change all the images at once in another task, since this need to be done for all of it. It makes more sense than making a single change now.

Reviewer 3: Well, indeed apologies, I hadn't realized it isn't a new image. You're 100% right!

   :align: center
   :alt: Capture Delay settings in Adyen

.. warning::
   If you set the Capture Amount Manually on Odoo but not on Adyen, Odoo won't be able to see it,
   your request for authorization only will result in an authorization and immediate capture on

>    Adyen side.

Reviewer 3: since you use an "if", use "would" instead of "will".
By the way, we try to avoid using the future tense for things that are always true --> that's the present simple.

>    Adyen side.

Author: Indeed, Adyen won't allow us to make a manual capture if we don't enable it, and on Odoo side, we don't have any way to see if the user enabled it or not. 
(sorry for the 3 unnecessary pushes for this, I changed it 3 times before coming back to your suggestion that's still the most understandable >.<)

.. warning::
   After **7 days**, the authorization can be **revoked** by the customer if it's still not

>    captured.

Reviewer 3: you can list multiple warnings with a list in a single warning admonition. Not a rule, but I think it's more readable.

>    captured.

Author: Oh, didn't think about that! That's nice!
